src/game/actionexecutor.cpp

```diff
@@ -66,6 +66,9 @@ void ActionExecutor::executeActions(Object &object, float dt) {
         case ActionType::StartConversation:
             executeStartConversation(static_cast<Creature &>(object), *dynamic_cast<StartConversationAction *>(action), dt);
             break;
+        case ActionType::AttackAction:
+            executeAttack(static_cast<Creature&>(object), *dynamic_cast<AttackAction*>(action), dt);
```

Comment:
Let's use the following code style (whitespace after type name, not before variable name):

```cpp
Creature &creature;
AttackAction *action;
```